src/style.css

```diff
@@ -1 +1,151 @@
-/* empty source */
+/* #region page */
+:root {
+  --min-width-container: 200px;
+  --content-card-margin-top: 16px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-weight: 400;
+  font-family: "Roboto", Arial, Helvetica, sans-serif ;
+  color: #060b35;
+}
+
+/* #endregion */
+
+/* #region cardSet */
+.card {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: var(--min-width-container);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+/* #endregion*/
+
+/* #region cardImage*/
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+/* #endregion*/
+
+/* #region cardDescription */
+.card__description {
+  align-self: center;
+  margin: 40px auto 0;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+.card__product-number {
+  align-self: flex-start;
+  margin-top: 4px;
+  margin-bottom: 0;
+  font-size: 10px;
+  color: #616070;
+}
+
+/* #endregion */
+
+/* #region cardReviews */
+
+.card__reviews-container {
+  display: flex;
+  align-self: flex-start;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 17px;
+  height: 16px;
+  width: 100%;
+}
+
+.card__reviews-count {
+  font-size: 10px;
+  text-align: center;
+  line-height: 14px;
+}
+
+.stars {
+  display: flex;
+  align-self: flex-start;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  display: inline-block;
+  width: 12px;
+  height: 12px;
+  margin: 2px 6px 2px 2px;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+/*#endregion*/
+
+/* #region price*/
+
+.card__price-container {
+  display: flex;
+  align-self: flex-start;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  line-height: 18px;
+  margin-top: 24px;
+}
+
+.card__price-title {
+  margin: 0;
+  color: #616070;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+.card__price-amount {
+  margin: 0;
+  font-weight: 700;
+  font-size: 16px;
+}
+
+/*#endregion*/
+
+/* #region button*/
+
+.card__button-link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 166px;
+  margin-top: var(--content-card-margin-top) ;
+  border: 1px solid transparent;
+  padding: 11px 0;
+  background: #00acdc;
+  border-radius: 5px;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.card__button-link:hover {
+  color: #00acdc;
+  background-color: #fff;
+  border: 1px solid #00acdc;
```

Comment:
You can just change the border-color on hover
![image](https://user-images.githubusercontent.com/96409563/221789630-2dd0f89b-1b3d-4b0c-9529-d8448b282d62.png)